addons/base_import/static/src/import_action/import_action.js

```diff
@@ -107,7 +107,10 @@ export class ImportAction extends Component {
 
     async onOptionChanged(name, value, fieldName = null) {
         this.model.block();
-        await this.model.setOption(name, value, fieldName);
+        const res = await this.model.setOption(name, value, fieldName);
+        if (res && res.file_length) {
```

Comment:
maybe you could do like in the handleFilesUpload, and do something like

```suggestion
        const { res, error } = await this.model.setOption(name, value, fieldName);
        if (!error) {
```
assuming the return values are the same (but you would also need to modify the empty return in `setOption` I guess)